\documentclass[10pt]{article}
%
%NSF biosketch
%
%updated 6/8/12 for IGERT grant
%
\usepackage{amsmath,epsfig,pstricks,xspace,graphpap,ifthen,natbib} 
\usepackage{setspace}
\usepackage{fullpage}
\usepackage{url}
%\usepackage[textwidth=6.5in,textheight=9in,tmargin=0.5in]{geometry}
\usepackage{fancyhdr}

%\lhead{Michael A. Gilchrist CV}
\rfoot{Gilchrist, CV \thepage}


\begin{document}


\begin{center}
  \begin{bfseries}
    { BIOGRAPHICAL SKETCH}\\
    Michael A. Gilchrist, Ph.D.
  \end{bfseries}
\end{center}

\subsection*{Professional Preparation}
\begin{tabular*}{\textwidth}{l@{\extracolsep{\fill}}l@{\extracolsep{\fill}}l@{\extracolsep{\fill}}l@{\extracolsep{\fill}}l}
Institution & Location & Program & Degree&Dates\\ \hline
University of California& Berkeley, CA& Environmental Sciences & B.A. & 1993\\
University of California& Santa Barbara, CA& Ecology, Evolutionary Biology& M.A.& 1996 \\
&&\hspace{10pt} \& Marine Sciences\\
Duke University& Durham, NC& Zoology& Ph.D.& 2001\\
Los Alamos National Laboratory& Los Alamos, NM & Theoretical Biology& Postdoc& 2001-2002\\
University of New Mexico& Albuquerque, NM& Biology& Postdoc&2002-2004
\end{tabular*}


\subsection*{Professional Appointments}
\begin{tabular}{lp{5.5in}}
2004-2010 & Assistant Professor, Dept.~of Ecology \& Evol.~Biology, University of Tennessee, Knoxville\\
2010-Present & Associate Professor, Dept.~of Ecology \& Evol.~Biology, University of Tennessee, Knoxville
\end{tabular}

%Removed via Research Office's suggestions
%\subsection*{Honors \& Fellowships}
%\begin{tabular}{ll}
%1989 - 1993& Mayo Foundation Scholarship, Mayo Clinic, Rochester, MN\\
%1993& R.E.U. Fellowship, University of Michigan Biological Station, Pellston, MI\\
%1994 - 1995& Regentâ€™s Fellowship, University of California, Santa Barbara\\
%1997& International Travel Grant, Duke University, Durham, NC\\
%1997& Grants in Aid of Research, Sigma Xi\\
%1998& Explorers Club Exploration Grant, The Explorers Club, New York, NY \\
%1999 - 2001& National Science Foundation Dissertation Improvement Grant\\
%2000 & Research Experience Fellowships for Young Foreign Researchers, Japanese \\
%&\hspace{10pt} Ministry of Education, Science, Sports, and Culture, Tokyo, Japan\\
%\end{tabular}


%Removed via Research Office's suggestions
%\subsection*{Professional Affiliations}
%Society of the American Naturalist, Society for the Study of Evolution, Society for Mathematical Biology  

\subsection*{Expertise}
%Phylogenetics Preproposal Jan 2013
I have extensive expertise in the development and analysis of complex mathematical models based on mechanistic descriptions of various biological processes such as pathogen dynamics within a host and the effects of translation errors on protein production.
%For example, I have developed and studied models that describe the dynamics of pathogen within a host affects its transmission between hosts and how nonsense errors during protein translation contribute to the evolution of codon usage bias.
I also have substantial experience parameterizing models by fitting them to available datasets, including genome scale sets of coding sequences.
Additionally, I have even used these parameterized models  to predict biologically meaningful parameters such as ribosome pausing times or gene expression levels.
I am comfortable using both frequentist and bayesian approaches, including the use of model selection frameworks such as AIC or DIC.
My previous work in population genetics and  phylogenetics and the effect of codon usage bias and gene expression levels on sequence evolution makes me particularly well suited to help achieve the research objectives of this project.


\subsection*{Related Products}
\begin{tabular}{ll}
  \hspace*{0.5in} &$^*$Graduate student co-author \\
   &  $^\dagger$Post-Doctorate co-author \\
\end{tabular}

\begin{enumerate}
\item Shah, P.$^*$ and \textbf{M.A.~Gilchrist} 2011. Explaining complex codon usage patterns with selection for translational efficiency, mutation bias, and genetic drift. \emph{PNAS} 18:10231-10236. %\url{doi: 10.1073/pnas.1016719108}

\item Shah, P.$^*$ and \textbf{M.A.~Gilchrist} 2010. Effect of Correlated tRNA Abundances on Translation Errors and Evolution of Codon Usage Bias. \emph{PLoS Genetics} 6(9): e1001128. %\url{http://dx.doi.org/10.1371%2Fjournal.pgen.1001128}.

\item \textbf{Gilchrist, M.A.}, P.~Shah$^*$, and  R.~Zaretzki. 2009.  Measuring and detecting molecular adaptation in codon usage against nonsense errors during protein translation.  \emph{Genetics} 183:1493-1505.


\item \textbf{Gilchrist, M.A.} 2007. Combining Models of Protein Translation and Population Genetics to Predict Protein Production Rates from Codon Usage Patterns. \emph{Molecular Biology and Evolution}. 24:2362-2373.


%\item \textbf{Gilchrist, M.A.}~and A. Wagner. 2006. A Model of Protein Translation Including Codon Usage Bias, Nonsense Errors, and Ribosome Recycling.  \emph{Journal of Theoretical Biology} 239:417-434.




\item Hickerson, M.J., \textbf{M.A.~Gilchrist}, and N.~Takebayashi. 2003. Calculating a Molecular Clock from Phylogeographic Data: Moments and Likelihood Estimators. \emph{Evolution} 57: 2216 -2225. 


%\item \textbf{Gilchrist, M.A.}, D.~L.~Sulsky, and A.~Pringle. 2006. Fitness and Optimal Life-History Strategies in Filamentous Fungi.  \emph{Evolution} 60:970-979.

  
\end{enumerate}

\subsection*{Other  Products} %choose 5
\begin{enumerate}
\item Roy$^\dagger$, B., J.N.~Vaughn$^*$, B-H Kim$^*$, F.~Zhou$^*$, \textbf{M.A.~Gilchrist}, and A.G.~Von Arnim. 2010. The h Subunit of eIF3 Helps to Maintain Reinitiation Competence during Translation of mRNAs Harboring Upstream Open Reading Frames.  \emph{RNA} 16: 748-761.

\item Zaretzki, R., \textbf{M.A.~Gilchrist},  W.M.~Briggs, and A. Armagan$.^\dagger$ 2010. Bias Correction and Bayesian Analysis of Aggregate Counts in SAGE Libraries. \emph{BMC Bioinformatics} 11: 72. 

\item \textbf{Gilchrist, M.A.}, L.A.~Salter, and A.~Wagner.  2004. A Statistical Framework for Combining and Interpreting Proteomic Datasets.   \emph{Bioinformatics} 20: 689-700

\item \textbf{Gilchrist, M.~A.},  H.~Qin$^\dagger$ and R.~Zaretzki. 2007.  Modeling SAGE tag formation and its effects on data interpretation within a Bayesian framework. \emph{{BMC} Bioinformatics}. 8:403.

%\item Coombs, D., \textbf{M.A.~Gilchrist}, and C.L. Ball$^*$. 2007.  Evaluating the Importance of Within- and Between-Host Selection Pressures in the Evolution of Chronic Pathogens. \emph{Theoretical Population Biology} 72: 576-591.

%\item Ball, C. L., \textbf{M.A. Gilchrist}, and D. Coombs. 2007. Modeling Within-Host Evolution of {HIV}: Mutation, Competition and Strain Replacement. \emph{Bulletin of Mathematical Biology} 69: 2361-2385.

\item \textbf{Gilchrist, M.A.}~and D.~Coombs. 2006. Evolution of Virulence: Interdependence, Constraints, and Selection using Nested Models.  \emph{Theoretical Population Biology} 63:145-153.

%\item \textbf{Gilchrist, M.A.}, D.~L.~Sulsky, and A.~Pringle. 2006. Fitness and Optimal Life-History Strategies in Filamentous Fungi.  \emph{Evolution} 60:970-979.

%\item \textbf{Gilchrist, M.A.}, D.~Coombs, and A.S.~Perelson. 2004. Optimizing Within-host Viral fitness: Infected Cell Lifespan and Virion Production Rate. \emph{Journal of Theoretical Biology} 229: 281-288

%\item Nelson, P.~W., \textbf{M.A.~Gilchrist}, M.~A., D.~Coombs, J.M.~Hyman, and A.S.~Perelson. 2004. An age-structured model of HIV infection that allows for variations in the production rate of viral particles and the death rate of productively infected cells. \emph{Mathematical Biosciences \& Engineering} 1, 267-288.

%\item Coombs, D.,  \textbf{M.A.~Gilchrist},  J.~Percus, and A.S.~Perelson. 2003. Optimal Viral Production. \emph{Bulletin of Mathematical Biology} 65: 1003-1023

%\item \textbf{Gilchrist, M.A.}~and A.~Sasaki. 2002.  Modeling host-parasite coevolution: a nested approach based on mechanistic models.   \emph{Journal of Theoretical Biology} 218: 289-308.


%\item \textbf{Gilchrist, M.A.}, and H.~F.~Nijhout. 2001.  Nonlinear Developmental Processes as Sources of Dominance. \emph{Genetics} 159: 423-432.







\end{enumerate}

\subsection*{Synergistic Activities}
%A list of up to *five* examples that demonstrate the broader impact of the individual~s professional and scholarly
%activities that focuses on the integration and transfer of knowledge as well as its creation. Examples could
%include, among others: innovations in teaching and training (e.g., development of curricular materials and
%pedagogical methods); contributions to the science of learning; development and/or refinement of research tools;
%computation methodologies, and algorithms for problem-solving; development of databases to support research
%and education; broadening the participation of groups underrepresented in science, mathematics, engineering
%and technology; and service to the scientific and engineering community outside of the individual~s immediate
%organization.
\begin{enumerate}
%\item Development of improved analysis method of Serial Analysis of Gene Expression data.
\item Development of various software packages including DatasetAssess for bayesian analysis and interpretation of high-throughput immuno-coprecipitation data and SEMPPR for predicting gene expression levels from a gene's codon usage patterns.
\item Faculty Member of the Genome Science \& Technology Program at the University of Tennesee, Knoxville (UTK).
\item Senior Personnel at the National  Institute for Mathematical and Biological Synthesis (NIMBioS) at UTK.
\item Member of NIMBioS's REU Leadership Team and Mentor for REU students 2009, 2012, \& 2013.
\item Mentor for NIMBioS Post-doctoral fellows Drs.~Juanjuan Chai and Gesham Magombedze.
%\item Participant in local science education advocacy group: GENESTN.
%\item Participant and co-organizer in UTK community outreach group: TN Darwin Days
\end{enumerate}

\ifthenelse{1=1}{ %as of 2013, collaborators and advisees are not included anymore

\subsection*{Collaborators \& Other Affiliations}
%A list of all persons in alphabetical order (including their current
%organizational affiliations) who are currently, or who have been collaborators or co-authors with the
%individual on a project, book, article, report, abstract or paper during the 48 months preceding the
%submission of the proposal. Also include those individuals who are currently or have been co-editors of
%a journal, compendium, or conference proceedings during the 24 months preceding the submission of
%the proposal. If there are no collaborators or co-editors to report, this should be so indicated.

\subsubsection*{Collaborators \& Co-Editors}
%Dr.~Daniel Coombs (U.~British Columbia),
Dr.~Zhulin Feng (Purdue),
Dr.~Sandra Halonen (Montana State Univ.) 
%Dr.~Alan Perelson (Los Alamos National Laboratory),
%Dr.~Hong Qin (Tuskegee U.),
%Dr.~Emi Shudo (Los Alamos National Laboratory),
%Dr.~Ethan White (Utah State U.), and 
%Dr.~Thandi Onami (UTK)
Dr.~Jian Huang (UTK),
Dr.~Laura Kubatko (Ohio State U.),
Dr.~Suzanne Lenhart (UTK),
Dr.~Dana Mordue (NY Medical College),
Dr.~George Ostrouchov (Oak Ridge National Laboratory),
Dr.~Premal Shah (U.~Penn),
Dr.~Yasuhiro Suzuki (U.~KY)
Dr.~Albrecht Von Arnim (UTK),
Dr.~Maud Lelu (UMN),
Dr.~Russell Zaretzki (UTK),
Dr.~Xiaopeng Zhao (UTK).



\subsubsection*{Graduate and Postdoctoral Advisors}
\paragraph*{Graduate Advisors:} 
%A list of the names of the individual~s own graduate advisor(s) and principal postdoctoral sponsor(s),
% and their current organizational affiliations.
MA: Dr.~Alice Alldredge (U.~California Santa Barbara), Ph.D.: Drs.~Janis Antonovics (U.~Virginia) and William G.~Wilson (Duke), Post-Doctoral: Drs.~Alan Perelson (Los Alamos National Laboratory) and Andreas Wagner (U.~of Zurich)
\paragraph*{Thesis Advisor and Postdoctoral Sponsor:}
%A list of all persons (including their
%organizational affiliations), with whom the individual has had an association as thesis advisor, or with
%whom the individual has had an association within the last five years as a postgraduate-scholar sponsor.
%The total number of graduate students advised and postdoctoral scholars sponsored also must be
%identified.
Ph.D. Students: Dr.~Premal Shah (Post-Doc at UPenn),  Dr.~Adam Sullivan (Zoll Medical), William P.~Howell (UTK),  Cedric Landerer (UTK) (4 of 4 advised).
Post-Doctoral Students: Dr.~Juanjuan Chai (NIMBioS), Dr.~Gesham Magombedze (NIMBioS) (2 of 4 advised).

%Dr.~Hong Qin (Assistant Professor, Spelman College)
%Dr.~Michael Saum (Assistant Professor, Georgia Gwinnette College) 
%(1 of 2 advised).

}{}

\flushright{ \tiny\today}
\end{document}
